tests/test_vcs_hg/code/Colon: name

```diff
@@ -1 +0,0 @@
-Filename has a colon
```

Comment:
I did not mean to commit that. I don't even know how that got there. I cloned the repo, switched to a new bookmark, edited the other file, git add'd that file, committed and pushed.